### PR TITLE
#252 batch table too

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes a few things to tidy up #252

- added some padding around the input of the editable row
- also made the input smaller and right-aligned. thought that it looked better for numeric input
- tweaked the number of items returned and the number per invoice. we were getting heaps of rows for the invoice, and not may items returned for the add item autocompleter
- fixed up the OK button on the add item modal, so that it actually adds the item :)

### CODE REVIEW

File: packages/invoices/src/OutboundShipment/DetailView/modals/ItemDetails.tsx [hunk 1]
```diff
@@ -90,13 +90,15 @@ export const ItemDetails: React.FC<ItemDetailsProps> = ({ item, onSubmit }) => {
     value: Item | null
   ) => {
     setSelectedItem(value);
-    setValue('id', value?.id || '');
-    setValue('code', value?.code || '');
-    setValue('name', value?.name || '');
-    trigger('id');
+    setValue('itemId', value?.id || '');
```
Comment (mark-prins) [inline packages/invoices/src/OutboundShipment/DetailView/modals/ItemDetails.tsx:93]: yeah, this is a bit hokey. just a temporary thing until the FEFO logic comes in.

File: packages/mock-server/src/data/data.ts [hunk 2]
```diff
@@ -88,7 +88,7 @@ const createInvoiceLines = (
   const invoiceLines: InvoiceLine[][] = [];
 
   invoices.forEach(invoice => {
-    takeRandomSubsetFrom(items).forEach(item => {
+    takeRandomSubsetFrom(items, 10).forEach(item => {
```
Comment (mark-prins) [inline packages/mock-server/src/data/data.ts:91]: easier to see when items are added if there aren't 100s of rows already 🤷‍♂️

File: packages/mock-server/src/data/data.ts [hunk 3]
```diff
@@ -128,7 +128,7 @@ const createInvoiceLines = (
 };
 
 const createItems = (
-  numberToCreate = randomInteger({ min: 10, max: 20 })
+  numberToCreate = randomInteger({ min: 250, max: 500 })
```
Comment (mark-prins) [inline packages/mock-server/src/data/data.ts:131]: was only getting five items in the list and it doesn't look very good.

File: packages/common/src/ui/layout/tables/columns/EditableQuantityColumn.tsx [hunk 4]
```diff
@@ -48,7 +48,11 @@ export const getEditableQuantityColumn = <
 
     return (
       <BasicTextInput
-        sx={{ maxHeight: 40 }}
+        sx={{
+          maxHeight: 40,
+          paddingTop: '3px',
+          '& .MuiInput-input': { textAlign: 'right', width: '75px' },
+        }}
```
Comment (joshxg) [inline packages/common/src/ui/layout/tables/columns/EditableQuantityColumn.tsx:55]: Think a good target for `NumericTextInput` soon - adding the `type="number"` n stuff

File: packages/invoices/src/OutboundShipment/DetailView/types.ts [hunk 5]
```diff
@@ -29,4 +30,8 @@ export type CustomerInvoiceAction =
       type: ActionType.SortBy;
       payload: { column: Column<ItemRow> };
     }
-  | CustomerInvoiceUpdateInvoice;
+  | CustomerInvoiceUpdateInvoice
+  | {
+      type: ActionType.AddLine;
+      payload: InvoiceLine;
+    };
```
Comment (joshxg) [inline packages/invoices/src/OutboundShipment/DetailView/types.ts:37]: Nit: pretty redundant but the convention is generally to use `payload: { invoiceLine }` heh

File: packages/common/src/ui/layout/tables/columns/EditableQuantityColumn.tsx (hunk 4 above)
Comment (mark-prins) [reply packages/common/src/ui/layout/tables/columns/EditableQuantityColumn.tsx:55]: yep!

**Pull request merged**